[16.0][imp] - sanitized bank account number instead of account number

badbole opened the pull request:
Use sanitized_acc_number instead of acc_number - so the wizard from module import_bank_statement_file gets correct values for comparison, as it is also using sinitized_acc_number

i-vyshnevska committed:
[14.0][MIG] account_statement_import_camt54

i-vyshnevska committed:
[MIG][14.0] account_statement_import_camt_oca

i-vyshnevska committed:
[IMP] add option to create addtional final line

luc-demeyer committed:
[14.0][FIX]camt import

StefanRijnhart committed:
[FIX] Fix same eror in camt54 module

StefanRijnhart committed:
[FIX] Fix same eror in camt54 module

OCA-git-bot committed:
account_statement_import_camt54 14.0.1.1.0

OCA-git-bot committed:
account_statement_import_camt 14.0.1.1.0

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 75.0% (9 of 12 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_camt
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_camt/nl/

NL66278 committed:
[IMP] Adapt account_bank_statement_import_online to 14.0

Also Rename account_bank_statement_import_online => account_statement_import_online

NL66278 committed:
[IMP] Adapt account_bank_statement_import_online to 14.0

Also Rename account_bank_statement_import_online => account_statement_import_online

alexis-via committed:
account_statement_import_online: usability improvement on form view of online.bank.statement.provider: avoid speading the information on 3 tabs. As there are just a few fields, let's keep all the information directly visible.

Fix warning on search view of online.bank.statement.provider
Fix some tests
Button on journal form view to access the wizard
The wizard now returns the bank statements of the journal(s)
Fix wired strings

NL66278 committed:
[FIX] account_statement_import_online: further improvements.

NL66278 committed:
[FIX] account_statement_import_online: further improvements.

oca-travis committed:
[UPD] Update account_statement_import_online.pot

oca-travis committed:
[UPD] Update account_statement_import_online_ponto.pot

Commit:
[FIX] account_bank_statement_import_txt_xlsx: import sheet notes

Commit:
[FIX] account_statement_import_online_ponto: fix vals dict to create a statement.

OCA-git-bot committed:
account_statement_import_online_ponto 14.0.1.0.1

Commit:
[IMP] account_statement_import_online: don't import empty statements.

oca-travis committed:
[UPD] Update account_statement_import_online.pot

OCA-git-bot committed:
account_statement_import_online 14.0.2.0.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_online/

primes2h committed:
[MIG] account_bank_statement_import_txt_xlsx: Migration to 14.0

[FIX] Mostly currency-related fixes

[IMP] sample_statement_en test: add dates on description

Co-authored-by: Marco Colombo <marco.colombo@gmail.com>

[UPD] Update account_statement_import_txt_xlsx.pot

Commit:
[FIX] account_statement_import_online_ponto: verify TLS

Not sure why, cert validation was disabled. That is insecure.

@Tecnativa TT31034

OCA-git-bot committed:
account_statement_import_online_ponto 14.0.1.0.2

Commit:
[FIX] fixing empty  assuming  is a good default value #422

primes2h committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 64.3% (56 of 87 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_txt_xlsx/it/

Translated using Weblate (Italian)

Currently translated at 73.5% (64 of 87 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_txt_xlsx/it/

Translated using Weblate (Italian)

Currently translated at 98.8% (86 of 87 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_txt_xlsx/it/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (87 of 87 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_txt_xlsx/it/

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 83.3% (10 of 12 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_camt
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_camt/it/

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 50.0% (5 of 10 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_camt54
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_camt54/it/

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 26.6% (24 of 90 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_online/it/

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 43.7% (7 of 16 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_online_ponto
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_online_ponto/it/

OCA-git-bot committed:
account_statement_import_online 14.0.2.0.1

JordiBForgeFlow committed:
[account_statement_import_txt_xlsx][imp] add migration scripts

NL66278 committed:
[IMP] account_statement_import_online[_ponto] improvements.

1. Find partner if ther already is a bank account with the right IBAN;
2. Store (and display) raw import data to help in problem determination;
3. Make it easy to extend the parsing of import data;
4. Apply some clean coding principles.

NL66278 committed:
[IMP] account_statement_import_online[_ponto] improvements.

1. Find partner if ther already is a bank account with the right IBAN;
2. Store (and display) raw import data to help in problem determination;
3. Make it easy to extend the parsing of import data;
4. Apply some clean coding principles.

NL66278 committed:
[IMP] *_online_ponto: code organisation

Check situation that should raise exception immediately, prevent ident on normal flow

NL66278 committed:
[FIX] *_online Improvements after review

NL66278 committed:
[FIX] *_online Improvements after review

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (87 of 87 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_txt_xlsx/es_AR/

account_statement_import_txt_xlsx 14.0.1.0.1

TeoGoddet committed:
[IMP] account_bank_statement_import_camt: more infos in narration

with this commit, the narration field is filled with infos find in the camt file such as reversal indicator, return reason, cheque number, ...
before only a banking ref was present.

The transaction type field is also filled.

yankinmax committed:
[MIG] account_statement_import_camt: Migration to 15.0

yankinmax committed:
[MIG] account_statement_import_camt54: Migration to 15.0

Commit:
Added translation using Weblate (Catalan)

Commit:
Translated using Weblate (Catalan)

Currently translated at 100.0% (16 of 16 strings)

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_online_ponto
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_online_ponto/ca/

StefanRijnhart committed:
[IMP] account_statement_import_camt: look for currency under Ntry

Encountered in a Camt.054 statement: currency listed under
/BkToCstmrDbtCdtNtfctn/Ntfctn/Ntry/Amt/@Ccy

oca-travis committed:
[UPD] Update account_statement_import_online.pot

OCA-git-bot committed:
account_statement_import_online_ponto 14.0.1.1.0

OCA-git-bot committed:
account_statement_import_online 14.0.2.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_online/

omar7r committed:
[REN] account_bank_statement_import_online_paypal: Renamed to account_statement_import_online_paypal

omar7r committed:
[MIG] account_statement_import_online_paypal: Migrated to 14.0

yajo committed:
[FIX] account_statement_import_online: use unittest.mock and don't import tests at install

Installing this module failed because `mock` couldn't be found. Indeed there was a migration problem.

However, I was not testing the module; just installing. At install, tests shouldn't be imported.

This file was being imported because the dummy model was expected to exist in database for tests. Using odoo-test-helper for that now.

@moduon MT-295

OCA-git-bot committed:
account_statement_import_online 14.0.2.1.1

norlinhenrik committed:
[IMP] account_statement_import_txt_xlsx: wizard: 2 amount columns, for IN and OUT

[IMP] account_statement_import_txt_xlsx: wizard: amount2_reverse boolean field, with tests

manuelregidor committed:
[MIG]account_statement_import_online: Migration to 15.0

manuelregidor committed:
[MIG]account_statement_import_online_ponto: Migration to 15.0

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update account_statement_import_online_ponto.pot

ikapasi-initos committed:
[MIG] account_bank_statement_import_online_paypal: Migration to 15.0

alexis-via committed:
Add module account_statement_import_base

The 2 modules account_statement_import_online and
account_statement_import depend on account_statement_import_base (and
not on each other) and share common code, in particular a hook to update
the statement line. So we can now have reconciliation modules that use
this hook and therefore work both on file import and online import. More
details on https://github.com/OCA/bank-statement-import/issues/481.

Improve bank statement line form view and journal form view.

alexis-via committed:
Add module account_statement_import_base

The 2 modules account_statement_import_online and
account_statement_import depend on account_statement_import_base (and
not on each other) and share common code, in particular a hook to update
the statement line. So we can now have reconciliation modules that use
this hook and therefore work both on file import and online import. More
details on https://github.com/OCA/bank-statement-import/issues/481.

Improve bank statement line form view and journal form view.

alexis-via committed:
Add module account_statement_import_base

The 2 modules account_statement_import_online and
account_statement_import depend on account_statement_import_base (and
not on each other) and share common code, in particular a hook to update
the statement line. So we can now have reconciliation modules that use
this hook and therefore work both on file import and online import. More
details on https://github.com/OCA/bank-statement-import/issues/481.

Improve bank statement line form view and journal form view.

alexey-pelykh committed:
[FIX] account_statement_import_online_paypal: tests

alexis-via committed:
Migration script field online_raw_data -> raw_data

Commit:
[UPD] Update account_statement_import_online.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-14.0/bank-statement-import-14.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-14-0/bank-statement-import-14-0-account_statement_import_online/

NL66278 committed:
[IMP] *_online_ponto: forward port 12.0 improvements

Separate retrieval of data from ponto (buffer data) and
creation of statements.

nicomacr committed:
[MIG] account_statement_import_txt_xlsx: Migration to 15.0

[UPD] Update account_statement_import_txt_xlsx.pot

[UPD] README.rst

zaoral committed:
[ADD] account_statement_import_txt_xlsx: show mapping error

If there is a problem parsing the file we are not showing the error
to the user, instead a generic message telling something is wrong
with the type of file is shown, and we only showing the error if
we are running unit tests.

In this case we think is good to show the error to the user, this
way they can check what is wrong and fix it in the sheet mapping.

* We are only showing the title of the error as a HINT, we think that is
  a good start to undestand what needs to change in the sheet mapping.
* The complete traceback error is shown in the log if we needed for more
  detail review from the technical teams.

zaoral committed:
[IMP] account_bank_statement_import_txt_xlsx: CSV

[IMP] add csv meta data management
[FIX] exclude footer meta data
[IMP] import separated credit/debit column file
[FIX] make comptatible with new version of multi_step_wizard module & add migration file
[FIX] all not provided value are handled in_parse_decimal method
[REF] Remove unnecessary \n

Renamed to account_statement_import_sheet_file

Co-authored-by: zaoral

Commit:
[IMP] account_statement_import_camt: black, isort, prettier

Commit:
[16.0][MIG] account_statement_import_camt: Migrate to version 16.0

JordiBForgeFlow committed:
[IMP] account_statement_import_txt_xlsx: Allow mapping with reference to column numbers, and concatenation
A new field 'File does not contain header line' is added in the Statement Sheet Mappings. If you set to True,
then you can map the columns by indicating in each field of the 'Columns' section the column number in the file.

We also allow to concatenate multiple columns in the file to a single column to a single field of the statement line.
You have to indicate the names of the columns separated by comma.

JordiBForgeFlow committed:
[IMP] account_statement_import_txt_xlsx: add tests

norlinhenrik committed:
[IMP] account_statement_import_txt_xlsx: amount_debit_column, amount_credit_column

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 25.5% (24 of 94 strings)

Translation: bank-statement-import-15.0/bank-statement-import-15.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-15-0/bank-statement-import-15-0-account_statement_import_online/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (87 of 87 strings)

Translation: bank-statement-import-15.0/bank-statement-import-15.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-15-0/bank-statement-import-15-0-account_statement_import_txt_xlsx/it/

alexis-via committed:
ponto: date field to use is now configurable

Add debug messages
Set raw_data in bank statement lines

NL66278 committed:
[IMP] *_online_ponto: improved data retrieval logic

With Ponto we can only retrieve data backwards in time. Therefore
we override the normal _pull method to create statements in descending
order of statement date.

NL66278 committed:
[IMP] *_online_ponto: add timeout to request get and post

NL66278 committed:
[MIG] Remove 14.0 migration files

NL66278 committed:
[MIG] account_statement_import_online: Migration to 16.0

NL66278 committed:
[MIG] account_statement_import_online_ponto: Migration to 16.0

luc-demeyer committed:
account_statement_import_camt - fix counterparty data

luc-demeyer committed:
fix camt unit tests

Commit:
[IMP] account_statement_import_camt54: black, isort, prettier

Commit:
[16.0][MIG] account_statement_import_camt54: Migrate to version 16.0

Commit:
[IMP] account_bank_statement_import_camt54: Add view for transfer_line field

JordiBForgeFlow committed:
[IMP] account_statement_import_txt_xlsx: Calculate final balance if not provided in the import file

[UPD] Update account_statement_import_txt_xlsx.pot

account_statement_import_txt_xlsx 15.0.2.0.0

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-15.0/bank-statement-import-15.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-15-0/bank-statement-import-15-0-account_statement_import_txt_xlsx/

MiquelRForgeFlow committed:
[FIX] account_statement_import_online: not require odoo_test_helper

pedrobaeza committed:
[FIX] account_statement_import_txt_xlsx: Don't try to parse if no mapping

account_statement_import_txt_xlsx 15.0.2.0.1

kwoychesko committed:
[FIX] account_statement_import_online_paypal: Update CONFIGURE, USAGE, and ROADMAP to current Odoo & PayPal menus/procedures/links.

(1) Odoo's menu has changed over the recent versions, and these changes help new users to better understand how to create a PayPal-related Bank Account/Journal and configuration for syncing with PayPal's servers.
(2) The order of CONFIGURE was changed to the logical flow of obtaining PayPal credentials first and then configuring Odoo to match.
(3) PayPal's URL for transaction definitions has changed and was updated in ROADMAP.

kwoychesko committed:
[FIX] account_statement_import_online_paypal: force observance of "Allow Empty Statements" provider setting

Empty statements should only be created if "Allow Empty Statements" is checked. This commit adds a check for the state of the allow_empty_statements field.

badbole committed:
[imp] - sanitized bank account number instead of account number

badbole commented:
@alexis-via could you check/validate  this one please

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (36 of 36 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/es/

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_ofx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_ofx/es/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 74.4% (73 of 98 strings)

Translation: bank-statement-import-15.0/bank-statement-import-15.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-15-0/bank-statement-import-15-0-account_statement_import_txt_xlsx/it/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (5 of 5 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_base
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_base/hr/

Gelo-fl committed:
Added translation using Weblate (Spanish)

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_base
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_base/es/

Commit:
Added translation using Weblate (French)

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (5 of 5 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_base
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_base/fr/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (36 of 36 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/fr/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (4 of 4 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_ofx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_ofx/fr/

alexis-via committed:
account_statement_import_file: fix string

Commit:
[UPD] Update account_statement_import_file.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 58.3% (21 of 36 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/pt_BR/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (36 of 36 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/pt_BR/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (4 of 4 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_ofx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_ofx/pt_BR/

pablohmontenegro committed:
[MIG] account_statement_import_txt_xlsx: Migration to 16.0

Co-authored-by: zaoral

pere-aquarian committed:
Added translation using Weblate (Spanish)

pere-aquarian committed:
Translated using Weblate (Spanish)

Currently translated at 98.9% (97 of 98 strings)

Translation: bank-statement-import-15.0/bank-statement-import-15.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-15-0/bank-statement-import-15-0-account_statement_import_txt_xlsx/es/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_base
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_base/pt_BR/

Commit:
[UPD] Update account_statement_import_camt.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

Commit:
[UPD] Update account_statement_import_camt54.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

ao-landoo committed:
[IMP] account_statement_import_file: Rename file_import selection value to file_import_oca

OCA-git-bot committed:
[UPD] addons table in README.md

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_camt
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_camt/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (36 of 36 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_camt54
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_camt54/es/

victoralmau committed:
[FIX] account_statement_import_txt_xlsx: Make tests resilient

- Depending on the installed set of modules, the company currency may
  be USD or EUR. If the second case, these tests will fail, so we make
  sure that the company currency is USD for our tests, doing the change
  by SQL, as there's a Python constraint that prevents it.

Not needed in v17 due to: odoo/odoo#107113.

account_statement_import_txt_xlsx 15.0.2.0.2

[UPD] Update account_statement_import_txt_xlsx.pot

account_statement_import_txt_xlsx 15.0.2.0.3

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-15.0/bank-statement-import-15.0-account_statement_import_txt_xlsx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-15-0/bank-statement-import-15-0-account_statement_import_txt_xlsx/

OCA-git-bot committed:
[UPD] README.rst

badbole commented:
@alexis-via did you have the time to check this? not sure why runboat/build is pending or how to start it...

bosd commented:
> why runboat/build is pending or how to start it...

You could rebase and (force) push to create a runboat.

astirpe committed:
[FIX] Group statement lines by period before update/create

astirpe committed:
[16.0][IMP] account_statement_import_online_ponto: retrieve partner

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_camt54
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_camt54/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_ofx
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_ofx/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_base
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_base/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_file/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_camt
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_camt/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (31 of 31 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_camt
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_camt/es/

stefan-tecnativa committed:
[IMP] account_statement_import_online_paypal: pre-commit stuff

stefan-tecnativa committed:
[MIG] account_statement_import_online_paypal: Migration to 16.0

Commit:
[UPD] Update account_statement_import_online.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_online/

Commit:
[UPD] Update account_statement_import_online_ponto.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update account_statement_import_online_paypal.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[IMP] update dotfiles

Commit:
[UPD] Update account_statement_import_online.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_online/

Commit:
[UPD] Update account_statement_import_sheet_file.pot

OCA-git-bot committed:
[BOT] post-merge updates

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (95 of 95 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_sheet_file
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_sheet_file/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (154 of 154 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_online_paypal
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_online_paypal/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (90 of 90 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_online
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_online/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (24 of 24 strings)

Translation: bank-statement-import-16.0/bank-statement-import-16.0-account_statement_import_online_ponto
Translate-URL: https://translation.odoo-community.org/projects/bank-statement-import-16-0/bank-statement-import-16-0-account_statement_import_online_ponto/es/
